pkg/convert/tf_state.go

```diff
@@ -160,7 +168,11 @@ func TranslateState(info il.ProviderInfoSource, path string) (*plugin.ConvertSta
 						if resourceInfo != nil {
 							pulumiType = resourceInfo.Tok.String()
 						} else {
-							return nil, fmt.Errorf("failed to get resource info for %q", tfType)
+							diagnostics = append(diagnostics, &hcl.Diagnostic{
```

Comment:
Wouldn't provider builds care about program convert rather than state convert?
But yes I suppose either (or both) could support a flag to make this an error.

Reply:
Oh whoops :) Sorry, you're absolutely right.